abci/src/codec.rs

```diff
@@ -146,6 +146,7 @@ where
         Ok(len) => len,
         // We've potentially only received a partial length delimiter
         Err(_) if src_len <= MAX_VARINT_LENGTH => return Ok(None),
+        // Err(e) => return Err(Error::decode(e)),
```

Comment:
For interest's sake, what's this line for?

Reply:
Don't know, it should probably be removed!